packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -186,8 +215,12 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
 
     position: absolute;
     bottom: 20px;
+    width: calc(100% - $mdc-textfield-icon-padding);
```

Comment:
This isn't working because `$mdc-textfield-icon-padding` is not resolving. Do we need to do something to force Sass to interpolate this within `calc`?

Reply:
Interpolation ala: `width: calc(100% - #{$mdc-textfield-icon-padding}) seems to work.